include/multipass/cli/argparser.h

```diff
@@ -75,8 +75,7 @@ class ArgParser
     bool help_requested;
     int verbosity_level{0};
 
-    std::ostream& cout;
-    std::ostream& cerr;
+    std::ostream &cout, &cerr;
```

Comment:
Quite some formatting changes in code that was not otherwise modified. Was this some IDE lapse?

Reply:
I merged the lines, clang-format did the rest. Will undo if you dislike

Reply:
Nope, that's fine by me. The only thing is that clang-formatting of untouched code adds a bit of noise to review, but it wasn't much here. In general, running clang-format on only the changes of a commit(-range) rather than whole files avoids that.